src/time-picker/index.tsx

```diff
@@ -12,6 +12,19 @@ import { createICacheMiddleware } from '@dojo/framework/core/middleware/icache';
 import { Keys } from '../common/util';
 import bundle from './nls/TimePicker';
 import i18n from '@dojo/framework/core/middleware/i18n';
+import { createResource, DataTemplate } from '@dojo/framework/core/resource';
+
+function createMemoryTemplate<S = void>(): DataTemplate<S> {
```

Comment:
Can this not be imported like the other createMemoryTemplate?

Reply:
this is an actual widget, the helper is in the examples dir until we create a better one in framework